public/Lab-Files/Lesson02/Challenges/images.html

```diff
@@ -0,0 +1,26 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <title>Images!</title>
+    
+</head>
+    <body>
+        <h1>The perfect trip</h1>
+        
+<p>I would go to Jamaica.. <img src="http://previews.123rf.com/images/goinyk/goinyk1504/goinyk150400071/38755384-Nice-asfalt-road-with-palm-trees-against-the-blue-sky-and-cloud-Stock-Photo.jpg"
+    alt="Jamaica Image" ></p>
+
+
+<p>I would see scenes like...<img src="http://previews.123rf.com/images/melnyk58/melnyk581506/melnyk58150600381/41462100-Fish-boat-on-the-paradise-beach-of-Jamaica-Stock-Photo-rico-puerto-beach.jpg"
+    alt="Views of Jamaica" >
+</p>
+
+
+<p>And do things like...EAT!<img src="http://previews.123rf.com/images/paulbrighton/paulbrighton1303/paulbrighton130300008/21928161-Jerk-Chicken-Jamaican-marinated-BBQ-chicken-served-with-grilled-pineapple-rice-and-peas-and-lime-wed-Stock-Photo.jpg"
+    alt="Jamaican food">
+</p>
+        
+        
+        
+    </body>
+    </html>
```

Comment:
Great job, I see no errors!
The only thing I suggest is that you work on spacing and indentation.